tests/unit/test_configuration.py

```diff
@@ -92,6 +97,20 @@ def test_settings_with_stage_from_envvar(self):
         assert "STAGE2_environment.py" == config.environment_file
         del os.environ["BEHAVE_STAGE"]
 
+    def test_settings_runner_class(self, capsys):
+        config = Configuration("")
+        assert BaseRunner == config.runner_class
+
+    def test_settings_runner_class_custom(self, capsys):
+        config = Configuration(["--runner-class=tests.unit.test_configuration.CustomTestRunner"])
+        assert CustomTestRunner == config.runner_class
+
+    def test_settings_runner_class_invalid(self, capsys):
+        with pytest.raises(SystemExit):
+            Configuration(["--runner-class=does.not.exits.Runner"])
+        captured = capsys.readouterr()
+        assert "No module named 'does.not.exits.Runner' was found." in captured.err
```

Comment:
You probably wanted to write `does.not.exist.Runner` in both occurrences above? -- Certainly not a bit deal.

Reply:
You're right.